main.go

```diff
@@ -39,6 +40,14 @@ func main() {
 		config.Clustering.RaftBootstrap = c.Bool("raft-bootstrap-seed")
 		config.Clustering.RaftBootstrapPeers = c.StringSlice("raft-bootstrap-peers")
 
+		// NATS server specified on the command line takes precedence
+		if natsServer := c.String("nats-server"); natsServer != "" {
+			if config.NATS.Servers != nil {
+				fmt.Fprintf(app.Writer, "WARNING: overwriting NATS server(s) specified in config file %#v with command line value %s\n", config.NATS.Servers, natsServer)
```

Comment:
We don't need this print since it's expected CLI flags will override config file settings.

Reply:
I went back & forth on this as it has bitten me in the past when someone else helpfully made changes to the command line in startup scripts. Less of an issue when an explicit config file cli option is required (versus implicit/'well-known' locations). I'll remove it.